### ISSUE TITLE
Radio button role, label is not announced during arrow key navigation. 

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#590]
<dl><dt>Description</dt><dd>Informative (static) content is not readable by a screen reader.
Radio button role, label is not announced during arrow key navigation using down/up keys. 
Elements  - 
Do you like pizza?
Yes , No , Only on Tuesdays

Do you like gyros?
Yes!! , Nope., Only on Fridays</dd><dt>Impact</dt><dd>Critical</dd><dt>Recommendation</dt><dd></dd><dt>Source code</dt><dd><pre><code><div role="radiogroup" aria-labelledby="pizza-label"><div class="Radio is--flex-row"><input type="radio" name="pizza" id="yes" value="yes"><label for="yes" class="Field__label">Yes</label><div aria-hidden="false" class="Icon Icon--radio-checked Radio__overlay"><svg xmlns="http://www.w3.org/2000/svg" overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" height="24" width="24"><path d="M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5zm0-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z" vector-effect="non-scaling-stroke" fill="currentColor"></path></svg></div></div><div class="Radio is--flex-row"><input type="radio" name="pizza" id="no" disabled="" value="no"><label for="no" class="Field__label Field__label--disabled">No</label><div aria-hidden="true" class="Icon Icon--radio-unchecked Radio__overlay Radio__overlay--disabled"><svg xmlns="http://www.w3.org/2000/svg" overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" height="24" width="24"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z" vector-effect="non-scaling-stroke" fill="currentColor"></path></svg></div></div><div class="Radio is--flex-row"><input type="radio" name="pizza" id="tuesday" value="tuesday" checked=""><label for="tuesday" class="Field__label">Only on Tuesdays</label><div aria-hidden="true" class="Icon Icon--radio-unchecked Radio__overlay"><svg xmlns="http://www.w3.org/2000/svg" overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" height="24" width="24"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z" vector-effect="non-scaling-stroke" fill="currentColor"></path></svg></div></div></div></code></pre></dd><dt>Checkpoint</dt><dd>Reading Order (1.3.2.a)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/RadioGroup</dd></dl>

**Comment by schne324** [comment]
Test that the group label is read out in NVDA for a radio group. NOTE: it is acceptable that the group label is only read out upon entering the group (it does not have to be read out for each radio button)
